srv/salt/state/miniconda.sls

```diff
@@ -1,26 +1,24 @@
 # File:/srv/salt/miniconda.sls
 
-download_miniconda_installer:
+miniconda_installed:
+  cmd.run:
+    - name: source ~/miniconda3/etc/profile.d/conda.sh && conda --version
+
+miniconda_installer_present:
   file.managed:
     - name: /home/centos/Miniconda3-latest-Linux-x86_64.sh
     - source: https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
     - skip_verify: True
-    - keep_source: True  # keep to avoid re-downloading.
+    - keep_source: True  # Keep to avoid re-downloading.
```

Comment:
If a miniconda installation is suppose to be once and then always present, then this download is sitting taking up space. Also if enough time passes and its time to install and updated miniconda version, this might cause salt to use the older version miniconda installer rather than pulling down the newest one.

Reply:
Fair enough, done.